doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1030 @@
+.. _ncs_release_notes_210:
+
+|NCS| v2.1.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+The feature is made available in its current state though the design and interfaces can change between release tags.
+The feature will also be labeled with "EXPERIMENTAL" in Kconfig files to indicate this status.
+Build warnings will be generated to indicate when features labeled EXPERIMENTAL are included in builds unless the Kconfig option :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` is disabled.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+The following list includes the summary of the most relevant changes introduced in this release.
+
+* Added :ref:`experimental support <software_maturity>` for the nRF7002 Wi-Fi 6 companion IC with samples.
+  See :ref:`nrf7002dk_nrf5340` and :ref:`wifi_samples` for more information.
+* Added experimental support for `Matter over Wi-Fi with nRF7002 <ug_matter_overview_architecture_integration>`.
+* Feature-completed experimental support for Matter over Thread.
+  It uses the :ref:`Thread v1.3 certified stack <thread_ug_supported_features_v13>` introduced in the |NCS| v2.0.2.
+* Added experimental support for setting the TX power envelope with FEM power control for Bluetooth® LE and 802.15.4.
+* Added experimental support for Microsoft Azure Embedded C SDK for the nRF91 Series.
+* Added :ref:`support <software_maturity>` for enhanced power optimized device management and nRF Cloud Location Services through LwM2M/CoAP with AVSystem for the nRF91 Series.
+* Added support for :ref:`nRF Cloud full modem FOTA <nrf9160_fota>` for the nRF91 Series.
+* Added experimental support for the Nordic Distance Toolbox (NDT) on the nRF5340 SoC with samples.
+* Added Find My support for the nRF5340 SoC.
+* Updated Trusted Firmware-M to v1.6, with additional flash and RAM overhead optimizations.
```

Comment:
```suggestion
* Updated Trusted Firmware-M to v1.6, with additional RAM overhead optimizations.
```
The Flash optimization was already done for 2.0.0 release

Reply:
This is in highlights, so I need to double-check with stakeholders.

Reply:
Confirmed and edited.

Reply:
Update on stakeholders opinion?

Reply:
"So we did not optimize flash consumption since last release? If this is the case, lets remove it. But if not, I would say its worth mentioning".